common/lib/container-definitions/src/loader.ts

```diff
@@ -252,20 +252,27 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
 
     /**
      * Boolean indicating whether the container is currently connected or not
+     * @deprecated - 0.58, This API will be removed in 0.60.0
+     * Use `connectionState.connected` instead
+     * See https://github.com/microsoft/FluidFramework/issues/9167 for context
      */
     readonly connected: boolean;
 
     /**
      * Dictates whether or not the current container will automatically attempt to reconnect to the delta stream
      * after receiving a disconnect event
      * @param reconnect - Boolean indicating if reconnect should automatically occur
-     * @alpha
+     * @deprecated - 0.58, This API will be removed in 0.60.0
```

Comment:
Can you also mark `setAutoReconnect` and `resume` as deprecated in the Container class itself?